parsons/utilities/oauth_api_connector.py

```diff
@@ -0,0 +1,84 @@
+from oauthlib.oauth2 import BackendApplicationClient
```

Comment:
16 lines in this file violate pep8 guidance for 79-character line length limits.